frontends/web/src/components/backups/restore.jsx

```diff
@@ -123,7 +123,6 @@ export default class Restore extends Component {
         selectedBackup,
         requireConfirmation,
     }, {
-        password,
```

Comment:
If not used in rendering, you can remove it from the state map and just store it directly (`this.password`)?